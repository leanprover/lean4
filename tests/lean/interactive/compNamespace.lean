namespace Foo
namespace LongNamespaceExample

def x := 10

#check LongN
          --^ textDocument/completion
end LongNamespaceExample

#check LongN
          --^ textDocument/completion
end Foo

#check Foo.
         --^ textDocument/completion

#check Foo.LongN
              --^ textDocument/completion
open Foo

#check LongN
          --^ textDocument/completion
