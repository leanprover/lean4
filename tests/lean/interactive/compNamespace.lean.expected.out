{"textDocument": {"uri": "file://compNamespace.lean"},
 "position": {"line": 5, "character": 12}}
{"items": [{"label": "LongNamespaceExample", "kind": 9, "detail": "namespace"}],
 "isIncomplete": true}
{"textDocument": {"uri": "file://compNamespace.lean"},
 "position": {"line": 9, "character": 12}}
{"items": [{"label": "LongNamespaceExample", "kind": 9, "detail": "namespace"}],
 "isIncomplete": true}
{"textDocument": {"uri": "file://compNamespace.lean"},
 "position": {"line": 13, "character": 11}}
{"items": [{"label": "LongNamespaceExample", "kind": 9, "detail": "namespace"}],
 "isIncomplete": true}
{"textDocument": {"uri": "file://compNamespace.lean"},
 "position": {"line": 16, "character": 16}}
{"items":
 [{"label": "Foo.LongNamespaceExample", "kind": 9, "detail": "namespace"}],
 "isIncomplete": true}
{"textDocument": {"uri": "file://compNamespace.lean"},
 "position": {"line": 20, "character": 12}}
{"items": [{"label": "LongNamespaceExample", "kind": 9, "detail": "namespace"}],
 "isIncomplete": true}
